packages/react-percy-storybook/src/getStaticAssets.js

```diff
@@ -1,16 +1,83 @@
 import path from 'path';
+import walk from 'walk';
 
 const fs = require('fs');
 
+// Some build assets we never want to upload.
+const SKIPPED_ASSETS = [
+    'index.html',
+    'iframe.html',
+    'favicon.ico',
+    'static/',
+    /\.map$/,
+    /\.log$/,
+    /\.DS_Store$/
+];
+const MAX_FILE_SIZE_BYTES = 15728640;  // 15MB.
+
+// Synchronously walk the build directory, read each file
+function gatherBuildResources(buildDir) {
```

Comment:
@fotinakis This method is slightly different, but with a bit of work we probably could, yes.